felix/bpf-gpl/globals.h

```diff
@@ -23,11 +23,12 @@ struct cali_tc_globals {
 
 enum cali_globals_flags {
 	/* CALI_GLOBALS_IPV6_ENABLED is set when IPv6 is enabled by Felix */
-	CALI_GLOBALS_IPV6_ENABLED 		= 0x00000001,
-	CALI_GLOBALS_RESERVED1 			= 0x00000002,
-	CALI_GLOBALS_RESERVED2 			= 0x00000004,
-	CALI_GLOBALS_RESERVED3 			= 0x00000008,
-	CALI_GLOBALS_RPF_STRICT_ENABLED = 0x00000010,
+	CALI_GLOBALS_IPV6_ENABLED 		    = 0x00000001,
+	CALI_GLOBALS_RESERVED1 			    = 0x00000002,
+	CALI_GLOBALS_RESERVED2 			    = 0x00000004,
+	CALI_GLOBALS_RESERVED3 			    = 0x00000008,
+	CALI_GLOBALS_RPF_OPTION_DISABLED    = 0x00000010,
```

Comment:
Wouldn't it to make more sense to default to disabled if the option is not  set?

```suggestion
	CALI_GLOBALS_RPF_OPTION_ENABLED    = 0x00000010,
```

Reply:
Implemented as suggested